dandischema/models.py

```diff
@@ -58,8 +58,19 @@
 )
 ASSET_UUID_PATTERN = r"^dandiasset:" + UUID_PATTERN
 VERSION_PATTERN = r"\d{6}/\d+\.\d+\.\d+"
-DANDI_DOI_PATTERN = rf"^10.(48324|80507)/dandi\.{VERSION_PATTERN}"
-DANDI_PUBID_PATTERN = rf"^DANDI:{VERSION_PATTERN}"
```

Comment:
TODO: below in the code we also have `DANDI:`, should probably be moved here into a dedicated variable...

Reply:
done